k8s/upgrades/1.0.0-1.1.0/example-cstor-spc-upgrade-job.yaml

```diff
@@ -6,35 +6,47 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
+  #
   #VERIFY that you have provided a unique name for this upgrade job.
```

Comment:
Can we have a space between # and comment

Reply:
Done